web/models.py

```
@@ -25,6 +25,67 @@
+    video_url = models.URLField(blank=True, help_text="Optional video pitch URL for the campaign")
+    image = models.ImageField(upload_to="campaign_images/", blank=True, help_text="Optional campaign image")
+    approved = models.BooleanField(default=True, help_text="Indicates whether the campaign is approved by moderators")
+    live = models.BooleanField(default=True, help_text="Indicates whether the campaign is live and accepting donations")
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        ordering = ["-created_at"]
+        verbose_name = "Crowdfunding Campaign"
+        verbose_name_plural = "Crowdfunding Campaigns"
+
+    def __str__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)